frontend/src/concepts/pipelines/content/configurePipelinesServer/ObjectStorageSection.tsx

```diff
@@ -47,7 +49,24 @@ export const ObjectStorageSection = ({
 
   return (
     <FormSection
-      title="Object storage connection"
+      title={
+        <Flex>
+          <FlexItem>Object storage connection</FlexItem>
+          {loaded && !!connections.length && (
+            <FlexItem align={{ default: 'alignRight' }}>
+              <Tooltip content="Populate the form with credentials from your selected data connection">
```

Comment:
nit: I don't think we are using the term "data connection" anymore. Maybe just "connection"

Reply:
done

Reply:
@manaswinidas  done